README.md

```diff
@@ -1,20 +1,82 @@
 openshift-login
 ===============
 
-A Jenkins plugin which lets you login to Jenkins with your account on your OpenShift installation. Focuses on enabling SSO for Jenkins within your cluster.
+A Jenkins plugin which lets you login to Jenkins with your account on an OpenShift installation.
 
 
-The primary scenario is using this plugin when Jenkins is running in a OpenShift pod, and the jobs with Jenkins operate against the
-same OpenShift cluster that Jenkins is running in.  In this scenario, no additional configuration is required, but you must be running agaist v1.4 of OpenShift/Origin (https://github.com/openshift/origin/tree/release-1.4).
+Primary scenario
+--------------------------------
 
-Otherwise, for development purposes, or for scenarios where the OpenShift pod defaults do not apply, the configuration parameters are:
+This plugin can function with no additional configuration within Jenkins, but you must be running in an OpenShift Pod and against v1.4 of OpenShift/Origin (https://github.com/openshift/origin/tree/release-1.4).
 
-* service account directory:  The directory to load service account information from. Three files are referenced:  'namespace', 'ca.crt', and 'token'. They correspond to the OpenShift project, certificate, and authentication token for the service account.
-* service account name:  override for the service account name used when authenticating users against OAuth (default derived from token / client secret)
-* server prefix:  URI for the OpenShift OAuth endpoint
-* redirect URL: URL for the OpenShift API server
-* client ID:  override for the ID for the OpenShift OAuth client (default derived from service account information)
-* client secret:  override for the service account token (to change permissions for the OAuth client during the OAuth authentication flows)
+When running against that level of OpenShift/origin, and the plugin installed in your Jenkins instance, the authentication mechanism (the "Security Realm") established within your Jenkins instance is as follows:
+
+* If running outside of an OpenShift Pod, then on start up default Jenkins authentication is used, irregardless of the setting of the environment variable `OPENSHIFT_ENABLE_OAUTH`.  If you are using the [OpenShift Jenkins image](https://github.com/openshift/jenkins), you log in with the user name `admin`, supplying the password specified by the `JENKINS_PASSWORD` environment variable set on the container. If you do not override `JENKINS_PASSWORD`, the default password for `admin` is `password`.  For any other pre-existing Jenkins installation, your credentials into that specific system are needed. 
```

Comment:
> On the admin ID and JENKINS_PASSWORD, I was unsure from our IRC exchange if you wanted that element moved as well, so I flipped a coin; I am certainly comfortable with moving it back, with some sort of generic preamble that if you are not using login plugn, then

yeah, honestly I think the other doc probably says enough about it even after your edits (the JENKINS_PASSWORD variable is described) but this is what i was hinting at when i said i'd leave that PR open for you to merge in case you wanted to revisit it. :)